.github/workflows/build.yaml

```diff
@@ -37,6 +37,16 @@ jobs:
           export SKIP_FLAGS_NON_UNIT_TESTS="-Dcheckstyle.skip -Dpmd.skip -Dcpd.skip -Dfindbugs.skip -Dspotbugs.skip"
           echo "SKIP_FLAGS_NON_UNIT_TESTS=$SKIP_FLAGS_NON_UNIT_TESTS" >> $GITHUB_ENV
           echo "SKIP_FLAGS_ALL_TESTS=$SKIP_FLAGS_NON_UNIT_TESTS -Dmaven.test.skip=true" >> $GITHUB_ENV
+          if [[ $GITHUB_EVENT_NAME != "pull_request" && $GITHUB_REF_NAME=="main" ]]; then
```

Comment:
Trying to remember here the case where `GITHUB_EVENT_NAME==pull_request` and `GITHUB_REF_NAME=="main"`. Why did we have to specify `!= pull_request`?

Reply:
Good point, I also realized that `$GITHUB_EVENT_NAME != "pull_request"` is not necessary, for PRs `GITHUB_REF_NAME` vaule is `pr_number/merge`;
so going to drop that!